src/Tickets/Admin/All_Tickets/List_Table.php

```diff
@@ -29,6 +30,71 @@ class List_Table extends WP_List_Table {
 	 */
 	protected $per_page_option;
 
+
+	/**
+	 * The template object.
+	 *
+	 * @var Tribe__Template
+	 */
+	protected $template;
```

Comment:
One thing i am not a fan about this class is that it has grown to do a lot more stuff than the List Table now :( Maybe some logic needs to be separated into different files.